tools/generate-baseline.js

```diff
@@ -0,0 +1,167 @@
+
+	const properties = {};
+	const propertyValues = {};
+	const atRules = {};
+	const types = {};
+	const selectors = {};
+
+	for (const [key, baseline] of Object.entries(features)) {
+		let match;
+
+		// property names
+		if ((match = cssPropertyPattern.exec(key)) !== null) {
+			properties[match.groups.property] = baselineIds.get(baseline);
+			continue;
+		}
+
+		// property values
+		if ((match = cssPropertyValuePattern.exec(key)) !== null) {
+			if (!propertyValues[match.groups.property]) {
+				propertyValues[match.groups.property] = {};
+			}
+			propertyValues[match.groups.property][match.groups.value] =
+				baselineIds.get(baseline);
+			continue;
+		}
+
+		// at-rules
+		if ((match = cssAtRulePattern.exec(key)) !== null) {
+			atRules[match.groups.atRule] = baselineIds.get(baseline);
+			continue;
+		}
+
+		// types
+		if ((match = cssTypePattern.exec(key)) !== null) {
+			types[match.groups.type] = baselineIds.get(baseline);
+			continue;
+		}
+
+		// selectors
+		if ((match = cssSelectorPattern.exec(key)) !== null) {
+			selectors[match.groups.selector] = baselineIds.get(baseline);
+			continue;
+		}
+	}
+
+	return {
+		properties,
+		propertyValues,
+		atRules,
+		types,
+		selectors,
+	};
+}
+
+//------------------------------------------------------------------------------
+// Main
+//------------------------------------------------------------------------------
+
+// create one object with all features then filter just on the css ones
+const allFeatures = Object.values(webFeatures).reduce(
+	(acc, entry) => Object.assign(acc, flattenCompatFeatures(entry)),
+	{},
+);
+const cssFeatures = extractCSSFeatures(
+	Object.fromEntries(
+		Object.entries(allFeatures).filter(([key]) => key.startsWith("css.")),
+	),
+);
+const featuresPath = "./src/data/baseline-data.js";
+
+// export each group separately as a Set, such as highProperties, lowProperties, etc.
+const code = `/**
+ * @fileoverview CSS features extracted from the web-features package.
+ * @author tools/generate-baseline.js
+ * 
+ * THIS FILE IS AUTOGENERATED. DO NOT MODIFY DIRECTLY.
+ */
+
+export const BASELINE_HIGH = ${BASELINE_HIGH};
+export const BASELINE_LOW = ${BASELINE_LOW};
+export const BASELINE_FALSE = ${BASELINE_FALSE};
+
+export const properties = new Map(${JSON.stringify(Object.entries(cssFeatures.properties), null, "\t")});
+export const atRules = new Map(${JSON.stringify(Object.entries(cssFeatures.atRules), null, "\t")});
+export const types = new Map(${JSON.stringify(Object.entries(cssFeatures.types), null, "\t")});
+export const selectors = new Map(${JSON.stringify(Object.entries(cssFeatures.selectors), null, "\t")});
+export const propertyValues = new Map([${Object.entries(
+	cssFeatures.propertyValues,
+).map(
+	([key, value]) =>
+		`["${key}", new Map(${JSON.stringify(Object.entries(value), null, "\t")})]`,
+)}]);
+`;
+
+fs.writeFileSync(
+	featuresPath,
+	await prettier.format(code, { filepath: featuresPath }),
+);
```

Comment:
```suggestion
// load prettier config
const prettierConfig = await prettier.resolveConfig(featuresPath);

fs.writeFileSync(
	featuresPath,
	await prettier.format(code, { filepath: featuresPath, ...prettierConfig }),
);
```

Like in `tools/update-rules-docs.js`. Formatting doesn't seem correct without this.